keyboards/momoka_ergo/config.h

```diff
@@ -17,36 +17,30 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 #pragma once
 
```

Comment:
This should be put back
```suggestion

#include "config_common.h"

```